samples/community/samuelIkoli/template.json

```diff
@@ -0,0 +1,104 @@
+{
+  "pageDimensions": [
+    630,
+    404
+  ],
+  "bubbleDimensions": [
+    20,
+    15
+  ],
+  "customLabels": {},
+  "fieldBlocks": {
+    "MCQBlock1": {
+      "fieldType": "QTYPE_MCQ5",
+      "origin": [
+        33,
+        6
+      ],
+      "fieldLabels": [
+        "q1",
+        "q2",
+        "q3",
+        "q4",
+        "q5",
+        "q6",
+        "q7",
+        "q8",
+        "q9",
+        "q10",
+        "q11",
+        "q12",
+        "q13",
+        "q14",
+        "q15",
+        "q16",
+        "q17",
+        "q18",
+        "q19",
+        "q20"
+      ],
+      "bubblesGap": 33,
+      "labelsGap": 20
+    },"MCQBlock2": {
```

Comment:
Can you please prettify the json file as well?

Reply:
Done